### ISSUE TITLE
Name for the Indian base in mars-sim 

### ISSUE THREAD

**Issue by shirishag75** [mars-sim/mars-sim#279]
Here are some of the alternatives that people have suggested for the Indian base in the game so far. This is purely from Indian standpoint - 

1. Maangalan 
2. भौमभवन  - BhomBhavan 
3. Marshal 
4. Base SANI 
5. बस्ती / Basti 

In the above 5 I do like 1. and 4.

**Comment by mokun** [comment]
That's cool ! 

> Base SANI

Does SANI have to be capitalized ?

> भौमभवन - BhomBhavan
> बस्ती / Basti

is it okay to have just "BhomBhavan" and "Basti" ?

**Comment by mokun** [comment]
So far, we have the following 5 settlement name choice for ISRO.

https://github.com/mars-sim/mars-sim/blob/ba32d20ffe3972a04e58c16a833000805b84ebe1/mars-sim-core/src/main/resources/xml/settlements.xml#L3913-L3918

Does it make sense to have them as names ?

**Comment by mokun** [comment]
Also, can you provide a list of Indian vehicle names ?

**Comment by mokun** [comment]
@shirishag75 ,

In the past, it was just a big pool of candidate vehicle names to pick from, regardless your country of origin and sponsor.

Now I'm going to make mars-sim to name a vehicle specifically according to its sponsor.

https://github.com/mars-sim/mars-sim/blob/f5c926bebc0b209e03d526c4ee9bda9c6db9f96c/mars-sim-core/src/main/resources/xml/vehicles.xml#L340-L407

**Comment by mokun** [comment]
@shirishag75 

Now, it looks like this :  

https://github.com/mars-sim/mars-sim/blob/e867d7ac36b1d4c890dcd5eb2ac81b65170237ee/mars-sim-core/src/main/resources/xml/vehicles.xml#L342-L486

**Comment by midhatdrops** [comment]
@mokun Hello! Want to help in this issue but don't know exactly how. Should i just add the names for rovers and settlements in the actual release? Could you guide me please ?

**Comment by mokun** [comment]
@bevans2000 Can we start pinning more issues as `good first issue` and `help wanted` ? 

Surely @midhatdrops can find some he's interested in contributing to mars-sim.

**Comment by midhatdrops** [comment]
@mokun there's an official list of languages supported by mars-sim ? From those i saw in the commits i only know English and a little bit of Japanese, but if i could i would be glad to add brazilian portuguese(my native lang) to settlement and vehicle names.

**Comment by bevans2000** [comment]
> @bevans2000 Can we start pinning more issues as `good first issue` and `help wanted` ?
> 
> Surely @midhatdrops can find some he's interested in contributing to mars-sim.

Good point, we have been lapse note highlighting good first tasks. Let me have a scan through the issue list.

@midhatdrops  As to helping. You could use the Scenario Editor to create your own Reporting Authority that has a Brazilian bias. This would allow you to create specific names. Then we can bundle that in to the release.
Or you  could look create a Portuguese version by creating a new set of language bindings.
Or if you want to do some Java development, then there will be some Issues tagged as Good First.

**Comment by mokun** [comment]
@midhatdrops 

We have since added Brazil to be a part of this venture of going to Mars and settling there. 

See mars-sim-core/src/main/resources/xml/country/country_brazil.xml.

Also find Brazil's mission agenda at mars-sim-core/src/main/resources/xml/governance.xml.

Feel free to provide your input and comment.

**Comment by mokun** [comment]
In [governance.xml](https://github.com/mars-sim/mars-sim/blob/master/mars-sim-core/src/main/resources/xml/governance.xml), 
```xml

	<!--  Reporting authorities. The code attribute must match
		the ReportingAuthorityType enum -->
	<authorities>
		<authority code="AEB" name="Agencia Espacial Brasileira" corporation="false"
					agenda="Space Tracking" gender-ratio="0.85">
			<country name="Brazil"/>
			<settlement-name name="Alcantara" />
			<settlement-name name="Antonina" />
			<settlement-name name="Antonio Prado" />
			<settlement-name name="Escola" />
			<settlement-name name="Florianopolis" />
			<settlement-name name="Morretes" />		
			<settlement-name name="Paraty" />
			<settlement-name name="Sao Sebastiao" />	

			<rover-name name="Agrale" />  
			<rover-name name="Lobini" />  
			<rover-name name="Marcopolo" />    
			<rover-name name="Neobus" />  
			<rover-name name="Engesa" />  
			<rover-name name="Troller" /> 
			<rover-name name="Puma" />  
			<rover-name name="Santa Matilde" /> 
			<rover-name name="Gaganyaan" /> 
			<rover-name name="Kuiper" /> 
			<rover-name name="Modelos" /> 
			<rover-name name="Bingo" /> 
			<rover-name name="Energia" /> 			
			<rover-name name="Embraer" /> 	
			<rover-name name="Visiona" /> 	
																		
		</authority>
